src/components/Calculator.js

```diff
@@ -1,46 +1,70 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 import './Calculator.css';
 
 class CalculatorInterface extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+
+    this.handleClicks = this.handleClicks.bind(this);
+  }
+
+  handleClicks(event) {
+    const TargetValue = event.target.value;
+    this.setState((state) => calculate({
+      next: state.next,
+      total: state.total,
+      operation: state.operation,
+    }, TargetValue));
   }
 
   render() {
+    const { total, next, operation } = this.state;
     return (
-
       <div className="container">
-        <div className="display" />
+        <div className="display">
+          <h1>
+            {total}
+
+            {operation}
+
+            {next}
+          </h1>
+        </div>
         <div className="btns">
           <div className="button-section">
-            <button className="button" type="button">AC</button>
-            <button className="button" type="button">+/-</button>
-            <button className="button" type="button">%</button>
-            <button className="button operator" type="button">/</button>
-            <button className="button" type="button">7</button>
-            <button className="button" type="button">8</button>
-            <button className="button" type="button">9</button>
-            <button className="button operator" type="button">x</button>
-            <button className="button" type="button">4</button>
-            <button className="button" type="button">5</button>
-            <button className="button" type="button">6</button>
-            <button className="button operator" type="button">-</button>
-            <button className="button" type="button">1</button>
-            <button className="button" type="button">2</button>
-            <button className="button" type="button">3</button>
-            <button className="button operator" type="button">+</button>
+            <button onClick={this.handleClicks} value="AC" className="button" type="button">AC</button>
+            <button onClick={this.handleClicks} value="+/-" className="button" type="button">+/-</button>
+            <button onClick={this.handleClicks} value="%" className="button" type="button">%</button>
+            <button onClick={this.handleClicks} value="/" className="button operator" type="button">/</button>
```

Comment:
- [ ] When I divide two numbers and click the _**=**_ button, the screen goes blank. Remember that in the _**operate.js**_file, the division symbol is represented by `÷` and not `/`. You would have to change the symbol and `value` to `÷` so that it can work. 👍